fastlane_core/spec/itunes_transporter_spec.rb

```diff
@@ -1107,9 +1126,64 @@ def xcrun_download_command(provider_short_name = nil, jwt: nil)
       end
     end
 
+    describe "with Xcode 14.x installed" do
```

Comment:
It's better to use `context` to describe situations.

```suggestion
    context "with Xcode 14.x installed" do
```